### DISCUSSION TITLE
Login Using Facebook

### DISCUSSION THREAD

**Discussion by Popoy-Dev** [supabase/supabase#2757] [Questions] [answered]
I added Facebook AUTH in my project, when I trying to login using another account  there is no AUTH user data inserting in my table.

**Comment by silentworks** [comment, marked as answer]
Can you provide more information as to what you have done? what you have provided here isn't sufficient for someone to help you out.

Also if your Facebook APP is still in developer mode, I think only the registered email address can be used to log into your Supabase app.
